latex/Research_Strategy.tex

```diff
@@ -450,7 +497,7 @@
 As in SAMPL3~\cite{muddana_sampl3_2012}, these acyclic CB[$n$]-type receptors introduce conformational complexity, and water interactions play a key role.
 Moreover, the presence of 4 anionic groups near the cavity will likely impact the balance between ion-dipole interactions and solvation of the free host.
 
-{\bf Subaim 2.2. Gibb deep cavity cavitands for host-guest studies} 
+\textbf{Subaim 2.2. Gibb deep cavity cavitands for host-guest studies} 
```

Comment:
But, people will still call the first two "octa acids" since they are acids and that is what they have always been called. So, basically, we have to use both names whether we want to or not. :(